components/web/icedtea-web/firefox-java.p5m

```diff
@@ -22,19 +22,36 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license="$(COMPONENT_LICENSE)"
 
-file usr/bin/javaws path=usr/jdk/instances/openjdk1.7.0/jre/bin/javaws mode=0555
-link path=usr/lib/firefox/browser/plugins/IcedTeaPlugin.so target=../../../IcedTeaPlugin.so
+file usr/bin/javaws path=usr/jdk/instances/openjdk1.8.0/jre/bin/javaws mode=0555
+link path=usr/lib/$(MACH64)/firefox/browser/plugins/IcedTeaPlugin.so target=../../../IcedTeaPlugin.so
 
-depend type=require fmri=pkg:/runtime/java/openjdk7
+depend type=require fmri=pkg:/runtime/java/openjdk8
 
 # Generated section:
+file path=usr/bin/itw-modularjdk.args
 file path=usr/bin/itweb-settings
 file path=usr/bin/javaws
 file path=usr/bin/policyeditor
-file path=usr/lib/IcedTeaPlugin.so mode=0555
+file path=usr/lib/$(MACH64)/IcedTeaPlugin.so
```

Comment:
Do I understand correctly that only user interface of icedtea is javaws, and IcedTeaPlugin can't be used by browser? So why do we ship it?

Reply:
Removed